pallets/template/Cargo.toml

```diff
@@ -5,7 +5,7 @@ description = "FRAME pallet template"
 edition = { workspace = true }
 license = { workspace = true }
 repository = { workspace = true }
-version = "0.1.0"
```

Comment:
no changes on this pallet indeed. I was also surprised by the bump, but parity did it. I thought it could be confusing for the users of this template to see inconsistent versioning between parity and us, hence I bumped it here as well

Reply:
My suggestion there would be just to remove template pallet. I do not think that it belongs to our repo because we are not templating the pallet development, but the runtime development.

Reply:
Fantastic! I agree as well 👍